docs/usage.md

```diff
@@ -52,35 +82,45 @@ You will need to create a samplesheet with information about the samples you wou
-| `sample`       | Custom sample name. This entry will be identical for multiple sequencing libraries/runs from the same sample. Spaces in sample names are automatically converted to underscores (`_`). |
-| `fastq_1`      | Full path to FastQ file for Illumina short reads 1. File has to be gzipped and have the extension ".fastq.gz" or ".fq.gz".                                                             |
-| `fastq_2`      | Full path to FastQ file for Illumina short reads 2. File has to be gzipped and have the extension ".fastq.gz" or ".fq.gz".                                                             |
-| `strandedness` | Sample strand-specificity. Must be one of `unstranded`, `forward` or `reverse`.                                                                                                        |
+| Column    | Description                                                                                                                                                                            |
+| --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `sample`  | Custom sample name. This entry will be identical for multiple sequencing libraries/runs from the same sample. Spaces in sample names are automatically converted to underscores (`_`). |
+| `fastq_1` | Full path to FastQ file for Illumina short reads 1. File has to be gzipped and have the extension ".fastq.gz" or ".fq.gz".                                                             |
+| `fastq_2` | Full path to FastQ file for Illumina short reads 2. File has to be gzipped and have the extension ".fastq.gz" or ".fq.gz".                                                             |
+| `bam`     | Full path to BAM file created with STAR and duplicate marked. File has to have the extension ".bam".                                                                                   |
+| `bai`     | Full path to index file of the BAM file. File has to have the extension ".bai".                                                                                                        |
+| `cram`    | Full path to CRAM file created with STAR and duplicate marked. File has to have the extension ".cram".                                                                                 |
+| `crai`    | Full path to index file of the CRAM file. File has to have the extension ".crai".                                                                                                      |
+| `vcf`     | Full path to vcf.gz file. File has to have the extension ".vcf.gz".                                                                                                                    |
```

Comment:
Is there also vcf input option? this wasnt in the previous description

Reply:
Yes it's possible to provide VCF files to be annotated to the pipeline. (I had to look it up myself wow, I didn't even know we had this :grin:). I'll update the docs